mysql-test/main/openssl.test

```diff
@@ -1,15 +1,13 @@
+# This is OpenSSL test.
 #
-# MDEV-6975 Implement TLS protocol
-#
-# test SSLv3 and TLSv1.2 ciphers when OpenSSL is restricted to SSLv3 or TLSv1.2
+# Test SSLv3 and TLSv1.2 ciphers when OpenSSL is restricted to SSLv3 or TLSv1.2
+# 
+# Related: MDEV-6975 Implement TLS protocol
```

Comment:
we have a standard test header,
```
#
# MDEV-xxx issue subject
#
```
please, don't change that.
This is a test specifically for `--tls-version` command line option of mariadb command line client

Reply:
The Jira title in question is `MDEV-6975 Implement TLS protocol`. This title does not reflect the purpose and contents of this test, so having it would be misleading. The fact that it was there to begin with sent many people debugging down the wrong path. An MDEV makes sense when the test is to prevent a specific bug from happening again. But when testing basic functionality (like if TLS works) choosing one of hundreds TLS related MDEVs as the "title" does not seem purposeful in documenting what the test is about.

Reply:
this is a bad MDEV title, as written by the user. Something I rewrite MDEV title to be better, in this case, unfortunately, I didn't. You can have the test file saying
```
#
# MDEV-6975 Implement TLS protocol
#
# despite the confusing title, MDEV-6975 is about allowing mariadb command line clients to
# restrict TLS to a specific protocol version, that is about implementing the --tls-protocol
# command line option. And this is what the following tests.
#
```